### DISCUSSION TITLE
Viewing stats of RFID use

### DISCUSSION THREAD

**Discussion by moerlien** [PatrikTrestik/homeassistant-solax-http#46] [Ideas]
The Solax Wallbox comes with 2 RFID Cards to lock the wallbox from unauthorized access and to track the charging processes independently with each card. So if you have two cars, you can see how much energy you have been charging in which one.

I was wondering if the data could also be tracked by homeassistant to create usage stats and other nice things.

![image](https://github.com/user-attachments/assets/f0e34c97-e495-4748-b3be-e39c825d6489)

**Comment by PatrikTrestik** [comment]
Communication with SolaX cloud is done by MQTT and server is hardcoded.
![image](https://github.com/user-attachments/assets/abf5f5d6-9764-4f29-ad4b-48833e3e7449)

There is option to hijack communication with DNS redirect hack, but it is not practical for normal use.
My idea is to watch if in some future version MQTT setting will be writable.

RFID events are sent to cloud by MQTT and I have no idea if and where it will be available on Modbus or API.

Cloud API has option to request data http hook. It can be option, but it is for other project 😃 

I'll close this discussion, because it is not possible to achieve by this integration.
